packages/js/product-editor/src/components/variations-table/inventory-menu-item/inventory-menu-item.tsx

```diff
@@ -233,6 +235,12 @@ export function InventoryMenuItem( {
 						>
 							{ __( 'Edit low stock threshold', 'woocommerce' ) }
 						</MenuItem>
+						<VariationQuickUpdateMenuItem.Slot
+							group={ 'inventory' }
+							supportsMultipleSelection={
+								supportsMultipleSelection
+							}
+						/>
```

Comment:
We need to pass through the `selection` and `onChange` so that fills can access that in their `onClick` handler and actually do something!

We should also pass  the`onClose` callbacks, as that will be needed too for extensions to close the menu when they have done their action!